src/Native/Common/pal_config.h.in

```diff
@@ -36,6 +36,9 @@
 #cmakedefine01 HAVE_TCP_VAR_H
 #cmakedefine01 HAVE_RT_MSGHDR
 #cmakedefine01 HAVE_LINUX_RTNETLINK_H
+#cmakedefine01 HAVE_INOTIFY_INIT
```

Comment:
I'll fold the 3 HAVE_ defines into 1.

@pgavlin - do you know how to make a compile/configure-time error just on Linux? (I am still learning cmake) How do I check for 'Linux'?